### PR TITLE
POL-454 No Root Account Access Key

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description
Adds a policy for finding if an AWS user has root access keys on their account.

### CODE REVIEW

**Comment by rshade** [comment]
We can probably put the CIS stuff in the readme also but I am not sure where

File: security/aws/iam_no_root_access_keys/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# AWS Root Access Keys
+
+## What it does
+
+The 'root' user account is the most privileged user in an AWS account. AWS Access Keys provide programmatic access to a given AWS account. It is recommended that all access keys associated with the 'root' user account be removed. This policy checks the Root account for access k. An incident is created if one or more are present.
```
Comment (rshade) [inline security/aws/iam_no_root_access_keys/README.md:5]: This policy checks the Root account for an access key. An incident is created if one or more access keys are present.

File: security/aws/iam_no_root_access_keys/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Provider tag value to match this policy: `aws` , `aws_sts`
+
+Required permissions in the provider:
+
+```javascript
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
```
Comment (rshade) [inline security/aws/iam_no_root_access_keys/README.md:36]: We can remove the sid

**Review by rshade — CHANGES_REQUESTED** [review]

File: security/aws/iam_no_root_access_keys/README.md (hunk 1 above)
Comment (XOmniverse) [reply security/aws/iam_no_root_access_keys/README.md:5]: Will fix and push update shortly.

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**